hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelineReportHandler.java

```diff
@@ -50,6 +52,8 @@
   private final Configuration conf;
   private final SCMSafeModeManager scmSafeModeManager;
   private final boolean pipelineAvailabilityCheck;
+  private Map<PipelineID, Map<UUID, ByteString>>
```

Comment:
Entries from this map need to be removed on pipeline destroy. I think we should maintain in this pipelineStateManger. @nandakumar131